html/generatedReport.php

```diff
@@ -509,23 +519,23 @@ class="icon-bar"></span>
 	<script type="text/javascript" src="../assets/js/jquery.js"></script>
 	<script type="text/javascript" src="../dist/js/bootstrap.min.js"></script>
 	<script type="text/javascript" src="//www.google.com/jsapi"></script>
-	
-	<script type="text/javascript" src="//canvg.googlecode.com/svn/trunk/canvg.js"></script>
-	<script type="text/javascript" src="//canvg.googlecode.com/svn/trunk/rgbcolor.js"></script>
-	<script type="text/javascript" src="//canvg.googlecode.com/svn/trunk/StackBlur.js"></script>
+
+  <script type="text/javascript" src="http://canvg.github.io/canvg/rgbcolor.js"></script>
```

Comment:
we need to change these back to `"//canvag...` so that way it works with http and https or just hardcode it to https, but we can't specify just http because when this site is running under https these are insecure assets.